ENCRYPTION.md

```diff
@@ -0,0 +1,113 @@
+#File Encryption in BCDA API
+
+When we deliver files via our data endpoint, we encrypt the files before sending them. You may wonder why we encrypt the files, since they are already being delivered via an encrypted channel (https). We do so for additional protection of all interested parties --- ourselves, you, and beneficiaries --- in the event of accidental or malicious activity like:
+
+* a malicious party steals your credentials or an access token. The party can make requests and download files, but can't read the data without being able to decrypt it.
+* a malicious party intercepts a file in transit. Again, they can't read the data without being able to decrypt it.
+
+These are just some examples of the ways in which data can go astray. In general, we should all err on the side of caution with protecting this data.
+
+Some best practices you can observe include 
+* guarding your client credentials and RSA private key assiduously 
+* storing them separately from each other
+* limiting access to only those who must use them.
+
+##How We Encrypt
+
+We encrypt the file as the last step in producing it, immediately before we return a final jobStatus (the one that has a body and no X-Progress header). Please see API.md for more on job status. 
+
+1. Generate a random 32 byte / 256 bit symmetric encryption key
+1. Generate a random nonce (also known as an Initialization Value, or IV)
```

Comment:
these can't all be the first step.

Reply:
that's how GFM does it. Always throws me, but you number them all one and then it magically renumbers them on render.

Reply:
oh, ok.  There is one below with 1,2,3,4 so I thought this was just an oversight

Reply:
Yeah, Markdown handles auto-numbering: https://daringfireball.net/projects/markdown/syntax#list

Personally speaking, I recommend numbering them by hand for shorter lists that aren't likely to change. For longer lists or ones that will be updated frequently, all `1.`s are good because they cut down on mistakes and diff noise.